testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/LabUserHelper.java

```diff
@@ -175,6 +192,16 @@ public static Credential getCredentials(LabUserQuery query) {
         return credential;
     }
 
+    public static AppInfo getAppInfo() {
+        LabAuthenticationHelper.getInstance().setupApiClientWithAccessToken();
+        AppApi api = new AppApi();
+        try {
+            return api.getAppByParam("cloud", "azurecloud", "azureadmultipleorgs", "yes", "yes").get(0);
```

Comment:
Would probably be better to accept each of these as method parameters instead of hard coding here